operate/qa/integration-tests/src/test/java/io/camunda/operate/schema/IndexOldSchemaValidatorIT.java

```diff
@@ -167,6 +168,51 @@ public void testOlderVersionsForIndex() {
     assertThat(indexSchemaValidator.olderVersionsForIndex(processIndex)).isEqualTo(olderVersions);
   }
 
+  @Test
+  public void testNewerVersionsForArchivedIndices() {
+    // Only older versions
+    final Set<String> givenIndexNames =
+        versionsOf(processIndex, olderVersions, "2022.01.01_10", "2022_01_01_10", "2022-01-01");
+    whenDatabaseClientReturnsIndexNames(givenIndexNames);
+    assertThat(indexSchemaValidator.newerVersionsForIndex(processIndex)).isEmpty();
+
+    // Only current version
+    whenDatabaseClientReturnsIndexNames(
+        versionsOf(
+            processIndex,
+            Set.of(processIndex.getVersion()),
+            "2022.01.01_10",
+            "2022_01_01_10",
```

Comment:
```suggestion
            "2022_01_01-10",
```

Looks like a copy-paste mistake?